stdlib/public/core/Arrays.swift.gyb

```diff
@@ -508,7 +508,7 @@ extension ${Self} : _ArrayType {
 
   /// Construct a ${Self} of `count` elements, each initialized to
   /// `repeatedValue`.
-  @swift3_migration(renamed="init(repeating:count:)")
+  @swift3_migration(message="Renamed to init(repeating:count:) and reordered parameters")
```

Comment:
It definitely can't; how would it know which parameters got reordered? Or would it assume any labels that didn't change are reordered rather than renamed?